hypershift-operator/controllers/hostedcluster/hostedcluster_controller.go

```diff
@@ -1073,8 +1109,8 @@ func reconcileCAPICluster(cluster *capiv1.Cluster, hcluster *hyperv1.HostedClust
 			Name:       hcp.Name,
 		},
 		InfrastructureRef: &corev1.ObjectReference{
-			APIVersion: "infrastructure.cluster.x-k8s.io/v1alpha4",
-			Kind:       "AWSCluster",
+			APIVersion: gvk.GroupVersion().String(),
```

Comment:
Is this actually working as expected? I'd suspect this will come empty.
Should APIVersion and Kind be set in (IBMCloud / AWS)Cluster?

Reply:
It works for IBM Cloud...... let me get you an example

Reply:
I was thinking the e2e test will pickup the AWS one but I could also go through and setup an AWS setup again to ensure that is the case.